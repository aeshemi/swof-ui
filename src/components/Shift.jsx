import React from 'react';
import { Media, Col } from 'react-bootstrap';

const Shift = props => (
  <Col md={6}>
    <Media.Heading>{props.name}</Media.Heading>
    <p>
      <strong>{props.shift}</strong>
    </p>
  </Col>
);

export default Shift;
