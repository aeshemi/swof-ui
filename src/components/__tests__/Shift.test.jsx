import Shift from '../Shift';

it('Shift renders correctly', () => {
  const wrapper = shallow(<Shift shift="Morning" name="Nap Nimmo" />);
  expect(wrapper).toMatchSnapshot();
});
