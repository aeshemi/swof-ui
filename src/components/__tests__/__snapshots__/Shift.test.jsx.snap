// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shift renders correctly 1`] = `
<Col
  bsClass="col"
  componentClass="div"
  md={6}
>
  <MediaHeading
    bsClass="media-heading"
    componentClass="h4"
  >
    Nap Nimmo
  </MediaHeading>
  <p>
    <strong>
      Morning
    </strong>
  </p>
</Col>
`;
